### PR TITLE
Some Editorial changes proposed by Chris for final check

### PR DESCRIPTION
**Pull request by billwuqin**

### CODE REVIEW

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 1]
```diff
@@ -423,37 +423,38 @@ Digital Twin Network Online Session {{DTPI2021}}, {{DTPI2022}}, and IEEE
...
-an indispensable part of the overall network system and provides a
-general architecture involving the whole life cycle of a "real"
-(typically, physical) network in the future, serving the application
-of network innovative technologies such as network planning, construction,
-maintenance and optimization, improving the automation and intelligence
-level of the network.
+an indispensable part of the overall network system, and can play an
+important role generally in architectures serving use cases across the
+whole life cycle of a "real" (typically, physical) network. Such use cases
+and applications span the range of network operations such as network
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:436]:
```suggestion
and applications span the range of network operations (e.g., network
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 2]
```diff
@@ -423,37 +423,38 @@ Digital Twin Network Online Session {{DTPI2021}}, {{DTPI2022}}, and IEEE
...
+planning, construction, maintenance and optimization, and improve the
+automation and intelligence level of the network.
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:438]:
```suggestion
automation and intelligence level of the network).
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 3]
```diff
@@ -872,14 +873,15 @@ orchestration/control services.
...
-components then work in composition with other functions or services
+components then work in composition with other logic, functions or services
 to deliver an overall functional architecture matching specific NDT use cases.
 
 For example: an NDT instance may be used as a core element of an intent-drive network controller.
 In such a case, an "outer" closed-loop (or, intent-assurance closed-loop) would detect
 gaps between target service objectives set by intents and actual observed service characteristics,
-find and proposed candidate mitigation solutions to soften the observed deviation, and drive the enforcement of the mitigation in the network.
-Finding such mitigations would rely on multiple "inner loops" that include an NDT: for example,
+find proposed candidate mitigation solutions to soften the observed deviation, and drive the
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:882]:
```suggestion
propose candidate mitigation solutions to soften the observed deviation, and drive the
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks.

File: draft-irtf-nmrg-network-digital-twin-arch.md (hunk 1 above)
Comment (billwuqin) [reply draft-irtf-nmrg-network-digital-twin-arch.md:436]: Good change

File: draft-irtf-nmrg-network-digital-twin-arch.md (hunk 3 above)
Comment (billwuqin) [reply draft-irtf-nmrg-network-digital-twin-arch.md:882]: Good change

File: draft-irtf-nmrg-network-digital-twin-arch.md (hunk 2 above)
Comment (billwuqin) [reply draft-irtf-nmrg-network-digital-twin-arch.md:438]: Med, Can you check again whether this right 
parentheses is needed？

**Pull request merged**